prow/config/presubmits.yaml

```diff
@@ -809,6 +809,310 @@ presubmits:
             memory: 4Gi
 
   maistra/istio:
+  - name: istio-integration-helm-2.4
+    trigger: (?m)^/test( | .* )integration\-helm,?($|\s.*)
+    rerun_command: /test integration-helm
+    skip_report: false
+    max_concurrency: 2
+    always_run: true
+    branches:
+      - ^maistra-2.4$
+    decorate: true
+    path_alias: istio.io/istio
+    spec:
+      containers:
+        - command:
+            - entrypoint
+            - prow/integ-suite-kind.sh
+            - test.integration.helm.kube.presubmit
+          env:
+            # - name: GOFLAGS
```

Comment:
nit: remove these comments

Reply:
Don't we want to bring it back once applying our changes?

Reply:
I don't think so. We are going to experiment removing this entirely for 2.4. Go is smart enough to detect when there's a vendor dir. If anything goes wrong, we can add it back later.

Reply:
Ok didn't know that. Removed in 97240a5